tmail-backend/mailbox/team-mailboxes/src/main/scala/com/linagora/tmail/team/TeamMailboxRepository.scala

```diff
@@ -79,6 +82,10 @@ class TeamMailboxRepositoryImpl @Inject()(mailboxManager: MailboxManager,
         case Some(validationFailure) => SMono.error(TeamMailboxNameConflictException(validationFailure.errorMessage))
         case None => createDefaultMailboxReliably(teamMailbox, session)
       })
+      .`then`(SFlux.fromIterable(teamMailboxCallbackSetScala)
+        .flatMap(_.teamMailboxAdded(teamMailbox), ReactorUtils.DEFAULT_CONCURRENCY)
+        .collectSeq()
+        .`then`(SMono.empty))
```

Comment:
```suggestion
   .`then`())
```

Reply:
The original code is to bypass the compiler. Otherwise we have:
```
 found   : reactor.core.scala.publisher.SMono[Unit]
 required: reactor.core.scala.publisher.SMono[Void]
        .`then`())
```